plugins/experimental/uri_signing/unit_tests/uri_signing_test.cc

```diff
@@ -34,6 +34,10 @@ extern "C" {
 #include "../match.h"
 #include "../config.h"
 
+#include "tscore/Version.h"
+
+AppVersionInfo appVersionInfo;
```

Comment:
Where is this used?

Reply:
i'm not sure. this was put in to resolve missing library symbols.